rosdep/python.yaml

```diff
@@ -6590,6 +6603,19 @@ python3-interpreter:
   rhel: [python3]
   slackware: [python3]
   ubuntu: [python3-minimal]
+python3-j1939-pip:
```

Comment:
Should we use an anchor/alias instead of repeated the same information for both Python 2 and 3? See [CONTRIBUTING](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1).

Reply:
@wjwwood What do you think?

Reply:
Oh yeah, I knew you could do that but I didn't realize it was recommended in the contributing file. It would be good to use the anchor.

Reply:
Thanks for the input, I didn't realize Anchors was preferred as well. I have pushed new anchor changes.